octoprint_octorelay/__init__.py

```diff
@@ -174,9 +55,9 @@ def on_after_startup(self):
 
         self._logger.info("--------------------------------------------")
         self._logger.info("start OctoRelay")
-        settings = self.get_settings_defaults()
+        settings = { **defaultSettings } # clone
```

Comment:
this variable is actually redundant, since we don't use it as a whole, only its index within the iteration